test/els_diagnostics_SUITE.erl

```diff
@@ -242,6 +245,64 @@ elvis(Config) ->
   end,
   ok.
 
+-spec escript(config()) -> ok.
+escript(Config) ->
+  Uri = ?config(diagnostics_escript_uri, Config),
+  ok = els_client:did_save(Uri),
+  {Method, Params} = wait_for_notification(),
+  ?assertEqual(<<"textDocument/publishDiagnostics">>, Method),
+  ?assert(maps:is_key(uri, Params)),
+  #{uri := Uri} = Params,
+  ?assert(maps:is_key(diagnostics, Params)),
+  #{diagnostics := Diagnostics} = Params,
+  ?assertEqual([], Diagnostics),
+  ok.
+
+-spec escript_warnings(config()) -> ok.
+escript_warnings(Config) ->
+  Uri = ?config(diagnostics_warnings_escript_uri, Config),
+  ok = els_client:did_save(Uri),
+  {Method, Params} = wait_for_notification(),
+  ?assertEqual(<<"textDocument/publishDiagnostics">>, Method),
+  ?assert(maps:is_key(uri, Params)),
+  #{uri := Uri} = Params,
+  ?assert(maps:is_key(diagnostics, Params)),
+  #{diagnostics := Diagnostics} = Params,
+  ?assertEqual(1, length(Diagnostics)),
```

Comment:
Why not just use `?assertMatch()` instead of asserting level by level? And then just pattern match? (This applies to the other test cases.
```suggestion
  ?assertMatch(#{uri := #{diagnostics := _}}, Params),
  #{uri := #{diagnostics := Diagnostics}} = Params,
```